pkg/network/tracer/tracer_test.go

```diff
@@ -1534,6 +1534,15 @@ func TestHTTPSViaOpenSSLIntegration(t *testing.T) {
 		t.Skip("HTTPS feature not available on pre 4.1.0 kernels")
 	}
 
+	kv, err := kernel.HostVersion()
```

Comment:
I think this entire test needs to be moved to `tracer_linux_test.go`. Using `kernel.HostVersion` doesn't work on Windows.

Reply:
Indeed, it seems that most calls to `kernel.HostVersion` are hidden behind `httpSupported` method. What do you think about adding such another method in the linux/windows specific versions so that we keep it the same fashion ?